docs: documentation overhaul (#1651)
GIScience/openrouteservice: docs/api-reference/endpoints/status/index.md

# Status Endpoint

The status endpoint os not available in the public API.

> The GET request http://localhost:8082/ors/v2/status (host and port might be different) returns basic information about the running instance:

Reviewer: ```suggestion
The GET request http://localhost:8082/ors/v2/status (host and port are dependent on the setup) returns basic information about the running instance:
```
